ui-common/src/components/TrafficViewer/EntriesList.tsx

```diff
@@ -187,29 +209,25 @@ export const EntriesList: React.FC<EntriesListProps> = ({
           />)}
         </ScrollableFeedVirtualized>
         <button type="button"
-                title="Fetch old records"
-                className={`${styles.btnOld} ${!scrollbarVisible && leftOffTop !== "" ? styles.showButton : styles.hideButton}`}
-                onClick={(_) => {
-                  trafficViewerApi.webSocket.close()
-                  getOldEntries();
-                }}>
-          <img alt="down" src={down}/>
+          title="Fetch old records"
+          className={`${styles.btnOld} ${!scrollbarVisible && Number.parseInt(leftOffTop) > 0 ? styles.showButton : styles.hideButton}`}
```

Comment:
@mertyildiran 
Is this condition can work?
showing the button only for leftOff is id and not "latest"

Reply:
We can't do `parseInt` because it's not always an integer anymore. [See](https://github.com/up9inc/mizu/pull/984#discussion_r847455298).